src/Tests/Moryx.Products.Management.Tests/Moryx.Products.Management.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net45</TargetFramework>
```

Comment:
Like other test projects, you directly use .NET5 as target framework also for tests. Have a look at your spaces: We normally use 4 spaces in xml instead of Tabs

Reply:
There is an option in studio to replace tabs with spaces.